examples/ansible/setup_libvirt_host.yml

```diff
@@ -4,10 +4,14 @@
 # A example playbook showing how the setup an ALP system as a libvirt host.
 #
 
+# use ansible-playbook setup_libvirt_host.yml -e "playbook_path=$(pwd)" -e "ansible_user=$(whoami)" -e "reboot_mode=auto" to enable auto reboot while running as root ensure ansible container is installed as root.
 
 - name: Setup ALP system as a libvirt host
   hosts: alphost
   vars:
+    ansible_playbook_basename: "setup_libvirt_host.yml"
```

Comment:
Facts/variables starting with `ansible_...` are usually reserved for the Ansible tool itself to use, and I don't see this being defined as a "standard" special variable/fact as documented here: https://docs.ansible.com/ansible/latest/reference_appendices/special_variables.html

I too have wished there was a fact/variable identifying the top-level playbook that is being run from inside an ansible task, but I don't believe there is a way to do so.

I would recommend changing this fact name to something more appropriate to the use case, e.g. alpreboot_resume_playbook, and either define it explicitly via set_fact action with an accompanying name: item that indicates why it is being defined, or as a "paramter" to the alp_reboot role, via a vars block when it is being included, to indicate the intended scope of where it is being used...